File: Server/FoodChooser/Models/FoodList.cs
using System;
using System.Collections.Generic;

namespace FoodChooser.Models
{
    public class FoodList
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string UserId { get; set; }
        public ICollection<FoodItem> Foods { get; set; }
    }
}

File: Client/app/models/foodList.ts

import { FoodItem } from './FoodItem';

export class FoodList {
    public Id: number;
    public Name: string;
    public Foods: FoodItem[];
}

export class FoodListWithImage {
    FoodListItem: FoodList;
    ImageString: string;

    constructor(list: FoodList, imageString: string) {
        this.FoodListItem = list;
        this.ImageString = imageString;
    }
}
